armi/reactor/composites.py

```diff
@@ -750,17 +750,24 @@ def getVolumeFractions(self):
         children = self.getChildren()
         volumes = [c.getVolume() for c in children]
         vol = sum(volumes)
-        return [(ci, vi / vol) for ci, vi in zip(children, volumes)]
+        if vol > 0.0:
+            volFracs = [(ci, vi / vol) for ci, vi in zip(children, volumes)]
+        else:
+            volFracs = [(ci, 0.0) for ci in children]
+        return volFracs
 
     def getVolumeFraction(self):
         """Return the volume fraction that this object takes up in its parent."""
+        if self.parent is None:
+            return 0.0
+
```

Comment:
undefined or None or error is more appropriate here. Let's talk more about how to justify this or accomplish what you need less surprisingly

Reply:
That makes sense. Probably should just raise an error.